arangod/Pregel/PregelFeature.cpp

```diff
@@ -442,3 +443,16 @@ void PregelFeature::handleWorkerRequest(TRI_vocbase_t& vocbase,
     w->aqlResult(outBuilder, withId);
   }
 }
+
+uint64_t PregelFeature::numberOfActiveConductors() const {
+  MUTEX_LOCKER(guard, _mutex);
+  uint64_t nr{0};
+  for (auto const& p : _conductors) {
+    std::shared_ptr<Conductor> c = p.second.second;
+    if (c->_state == ExecutionState::RUNNING ||
+        c->_state == ExecutionState::STORING) {
```

Comment:
I think we also need to count all conductors in state `DEFAULT`. `DEFAULT` is the initial conductor state and will be still set when the conductor is inserted into the PregelFeature's map of conductors.
```suggestion
    if (c->_state == ExecutionState::DEFAULT ||
        c->_state == ExecutionState::RUNNING ||
        c->_state == ExecutionState::STORING) {
```

Reply:
Good catch!